.babelrc

```diff
@@ -1,3 +1,8 @@
 {
-  "presets": ["@babel/preset-env"]
+  "presets": ["@babel/preset-env"],
+  "plugins": [
+      [
+        "@babel/plugin-proposal-class-properties"
```

Comment:
From the doc page (https://babeljs.io/docs/en/babel-plugin-proposal-class-properties): "NOTE: This plugin is included in @babel/preset-env"

So I'm not sure this is needed